src/Sylius/Bundle/AdminBundle/Resources/views/Channel/_form_content.html.twig

```diff
@@ -0,0 +1,93 @@
+{{ form_errors(form) }}
+
+<div class="ui two column stackable grid">
+    {{ sylius_template_event('sylius.admin.channel.before_form', _context) }}
```

Comment:
Can you check if the "form" variable is present inside of _context? Also, if some additional attribute is required to call it, lets add it explicitly, to be sure people may use it as in normal form template

E.g
<img width="553" alt="image" src="https://user-images.githubusercontent.com/6213903/160094551-1c87d37b-eb92-4b67-992a-26b52916218e.png">

Reply:
yup, that could be done like this, but in sylius plus we are using more than just forms, this way we allow user to use any variable in that event. I think its more customize friendly than sending just the form.

Reply:
But is the form available straight from the template or behind some additional variable? I

Reply:
Directly.

Reply:
Here's some examples of it being used it other places: https://github.com/Sylius/Sylius/search?l=Twig&q=_context